.github/workflows/continuous-integration-workflow.yml

```diff
@@ -5,16 +5,16 @@ on: [push]
 jobs:
   tests:
     name: "Python ${{ matrix.python-version }}"
-    runs-on: windows-2019
+    runs-on: windows-2022
     strategy:
       matrix:
-        python-version: ["3.7"]
+        python-version: ["3.9"]
```

Comment:
I have a PR open that includes some changes to the Windows CI (#48). I'm not convinced that `setup-python` works on Windows since it looks like `glpk` from conda pulls in python 3.12. In #48, I've removed the matrix entirely and just let conda pull in whatever python it needs/wants.